modules/installation-configuration-parameters.adoc

```diff
@@ -2642,6 +2672,61 @@ Additional VMware vSphere configuration parameters are described in the followin
         networks:
 |Lists any network in the vCenter instance that contains the virtual IP addresses and DNS records that you configured.
 |String
+ifdef::agent[]
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        computeCluster:
+|The path to the vSphere compute cluster.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        datacenter:
+|Lists and defines the datacenters where {product-title} virtual machines (VMs) operate.
+The list of datacenters must match the list of datacenters specified in the `vcenters` field.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        datastore:
+|The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
+[IMPORTANT]
+====
+You can specify the path of any datastore that exists in a datastore cluster.
+By default, Storage vMotion is automatically enabled for a datastore cluster.
+Red{nbsp}Hat does not support Storage vMotion, so you must disable Storage vMotion to avoid data loss issues for your {product-title} cluster.
+
+If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file.
+For more information, see "VMware vSphere region and zone enablement".
+====
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        resourcePool:
+|Optional. The absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`.
+If you do not specify a value, resources are installed in the root of the cluster, for example `/example_datacenter/host/example_cluster/Resources`.
+|String
+
+|platform:
+  vsphere:
+    failureDomains:
+      topology:
+        folder:
+|Optional. The absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`.
+If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID.
+If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass` object, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
```

Comment:
For agent-based installations, users must specify the folder where the VMs are created because the installer does not create the VMs in this scenario.

Reply:
If I'm interpreting this right, I'll get rid of line 2726 about "not providing the value". I'm assuming the line below it can stay and that this parameter is still Optional for ABI, but let me know if that's not the case.